carrier-io/Jenkinsfile

```diff
@@ -106,13 +106,14 @@ node {
 
 			stage("Send Notification") {
 				sendNotification(params.testName, params.testType, params.users.toInteger(), params.reportingInstanceUrl, params.emailsList, params.galloperTaskIdForNotifications)
-			}				
+			}
+			
+			stage('Shut down load generators and FOLIO environment') {
+			    stopMonitoringTask(params.targetCluster, params.targetEnv)
```

Comment:
It can't do stopMonitoringTask from the community's environment.  The activity of stop monitoring tasks involves issuing a command on a server that runs in the same environment as the FOLIO nodes to shut down the Telegraf tasks, but since we're in a cross-environment situation this call will not work.

Also, the stopMonitoringTask function is not defined.